modules/Trackers/Tracker.php

```diff
@@ -110,14 +110,11 @@ function get_recently_viewed($user_id, $modules = '')
 			$breadCrumb = $_SESSION['breadCrumbs'];
 	        $module_query = '';
 	        if(!empty($modules)) {
-	           $history_max_viewed = 10;
 	           $module_query = is_array($modules) ? ' AND module_name IN (\'' . implode("','" , $modules) . '\')' :  ' AND module_name = \'' . $modules . '\'';
-	        } else {
-	           $history_max_viewed = (!empty($GLOBALS['sugar_config']['history_max_viewed']))? $GLOBALS['sugar_config']['history_max_viewed'] : 50;
 	        }
 
-	        $query = 'SELECT item_id, item_summary, module_name, id FROM ' . $this->table_name . ' WHERE id = (SELECT MAX(id) as id FROM ' . $this->table_name . ' WHERE user_id = \'' . $user_id . '\' AND deleted = 0 AND visible = 1' . $module_query . ')';
-	        $result = $this->db->limitQuery($query,0,$history_max_viewed,true,$query);
+	        $query = 'SELECT item_id, item_summary, module_name, id FROM ' . $this->table_name . ' WHERE user_id = \'' . $user_id . '\' AND deleted = 0 AND visible = 1' . $module_query . ' ORDER BY id DESC ';
```

Comment:
Please look over queries taking into consideration SQL Injection - using the quoted() function where appropriate.

Reply:
I do not see that there ist more SQL injection possible than before. But feel free to adjust this.